snuba/query/processors/transaction_column_processor.py

```diff
@@ -0,0 +1,60 @@
+from snuba.query.expressions import Column, Expression, FunctionCall, Literal
+from snuba.query.query import Query
+from snuba.query.query_processor import QueryProcessor
+from snuba.request.request_settings import RequestSettings
+
+
+class TransactionColumnProcessor(QueryProcessor):
+    """
+    Mimics the old column_expr functionality
+    """
+
+    def process_query(self, query: Query, request_settings: RequestSettings) -> None:
+        def process_column(exp: Expression) -> Expression:
+            if isinstance(exp, Column):
+                # TODO remove these casts when clickhouse-driver is >= 0.0.19
```

Comment:
Clickhouse driver is now 0.1.1. Which means you probably should remove these ip_address_v4 and ip_address_v6.